java/SerializationUtilities/events/APIGatewayEvent.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Is this file actually used?

Reply:
Right now it is not used by SAM template. I've added it for local testing purpose. Let me know if it does not make any sense - I will remove it. 

As far as I understand events from other examples (e.g. for Validation) are also not used - am I right?

Reply:
OK, no problem. We can keep them.